From a786cecc0104792d26d82d97a346a3175d452d71 Mon Sep 17 00:00:00 2001
From: Oleg Kolosov <bazurbat@gmail.com>
Date: Tue, 4 Apr 2017 11:54:37 +0200
Subject: [PATCH] Add USE_DLT option

This allows to disable DLT even if it is found.
---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b62a926..692f4c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,6 +68,9 @@ message(STATUS "USE_FILE is set to value: ${USE_FILE}")
 OPTION(USE_CONSOLE "Set to OFF to disable console logging" OFF )
 message(STATUS "USE_CONSOLE is set to value: ${USE_CONSOLE}")
 
+OPTION(USE_DLT "Enable DLT" OFF)
+message(STATUS "USE_DLT is set to value: ${USE_DLT}")
+
 # Make relative paths absolute (needed later on)
 foreach(p LIB INCLUDE CMAKE)
   set(var INSTALL_${p}_DIR)
@@ -104,10 +107,12 @@ ENDIF(MSVC)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DCOMMONAPI_LOGLEVEL=COMMONAPI_LOGLEVEL_${MAX_LOG_LEVEL}")
  
 FIND_PACKAGE(PkgConfig)
+IF(USE_DLT)
 pkg_check_modules(DLT "automotive-dlt >= 2.11")
 IF(DLT_FOUND) 
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_DLT")
 ENDIF(DLT_FOUND)
+ENDIF(USE_DLT)
 
 ##############################################################################
 
-- 
2.10.2

